src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6115,13 +6303,14 @@ export class StudyViewPageStore
         }
 
         if (!_.isEmpty(this.structuralVariantProfiles.result)) {
-            const uniqueKey = getUniqueKeyFromMolecularProfileIds(
+            let uniqueKey = getUniqueKeyFromMolecularProfileIds(
```

Comment:
it seems odd that we make a uniqueKey, use it at 6312, then, conditionally update it at line 6323.

Reply:
if in fact that's the intention, then it's more like we are creating 2 independent keys, and the code should reflect that